packages/react-composites/tests/browser/chat/ErrorBar.test.ts

```diff
@@ -18,18 +13,6 @@ test.describe('ErrorBar is shown correctly', async () => {
     await chatTestSetup({ pages, users, serverUrl });
   });
 
-  test('not shown when nothing is wrong', async ({ serverUrl, users, page }) => {
```

Comment:
am I missing something here? It seems like we only wanted to create new message tests but we are impacting the errorbar tests? and snapshots?

Reply:
The first test in ErrorBar is a test with no error bar with one message sent. I figure it was the simplest test in all of the Chat UI tests.

Reply:
It would be good for us to be explicit about the tests that are run with the fake adapter and which ones arent. The error bar was just altered but it looked like it should have just been a normal test and not a fake adapter test.